File: 1.两数之和/双重循环法/两数之和.py
#会超出时间限制
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        result = list()
        for i in range(len(nums)-1):
            for j in range(i+1,len(nums)):
                if target == nums[i] + nums[j]:
                    result.append(i)
                    result.append(j)
        return result

File: 1.两数之和/双重循环法/两数之和.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        int len;
        vector<int> a;
        len=nums.size();
        for(int i=0;i<len-1;i++)
            for(int j=i+1;j<len;j++)
            {
                if(nums[i]+nums[j]==target)
                {
                    a.push_back(i);
                    a.push_back(j);
                    return a;
                }
            }
        return a;
    } 
};